analyzers/its/expected/Net5/Net5--net5.0-S2325.json

```diff
@@ -509,6 +509,19 @@
 },
 {
 "id":  "S2325",
+"message":  "Make 'Method' a static method.",
+"location":  {
+"uri":  "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/sources/Net5/Net5/S4201.cs#L11",
```

Comment:
Tried it locally, it is correct.
I am not sure why we did not find it beforehand.
https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/sources/Net5/Net5/S4201.cs#L11

Reply:
It's a side effect of filtering out the name portion of named parameters that you just introduced.
It was a FN earlier because of [this line](https://github.com/SonarSource/sonar-dotnet/blob/48f9399972fefba70fca547dcfead9a1e0ecb0ae/analyzers/its/sources/Net5/Net5/S4201.cs#L28). The pattern matching contains a `NamedSyntax` node.
Well done!
It's not perfect, this is still a FN:
```cs
public class MyClass
{
    public int Prop { get; set; }

    public MyClass Create() // FN - because of the property initialization
    {
        return new MyClass { Prop = 42 };
    }
}
```
I'll leave it to you whether to fix it here or in a separate PR. If the latter, then please add a reproducer for it.